### PR TITLE
SONAR-7036 Update password of technical user

### PR DESCRIPTION
**Pull request by julienlancelot**
SONAR-7036 Update password of technical user

When using an external authentication (like LDAP), it's now possible possible to update the password of technical accounts

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/user/UserUpdater.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 import com.google.common.collect.Iterables;
 import java.net.HttpURLConnection;
 import java.security.SecureRandom;
+import java.util.ArrayList;
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/user/UserUpdater.java]: MINOR Remove this unused import 'java.util.ArrayList'. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)

File: server/sonar-server/src/main/java/org/sonar/server/user/UserUpdater.java [hunk 2]
```diff
@@ -26,6 +26,7 @@
...
 import java.util.Arrays;
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/user/UserUpdater.java]: MINOR Remove this unused import 'java.util.Arrays'. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Comment by sns-seb** [comment]
LGTM

**Pull request merged**